<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>878</width>
    <height>307</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Умножение матриц</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout_8">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <widget class="QLabel" name="lbLeftRows">
            <property name="text">
             <string>Строки</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSpinBox" name="sbLeftRows">
            <property name="minimum">
             <number>1</number>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <widget class="QLabel" name="lbLeftColumns">
            <property name="text">
             <string>Столбцы</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSpinBox" name="sbLeftColumns">
            <property name="minimum">
             <number>1</number>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QTableView" name="tvLeftMatrix"/>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QLabel" name="lbMult">
      <property name="font">
       <font>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="text">
       <string>X</string>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_6">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_4">
          <item>
           <widget class="QLabel" name="lbRightRows">
            <property name="text">
             <string>Строки</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSpinBox" name="sbRightRows">
            <property name="minimum">
             <number>1</number>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_5">
          <item>
           <widget class="QLabel" name="lbRightColumns">
            <property name="text">
             <string>Столбцы</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSpinBox" name="sbRightColumns">
            <property name="minimum">
             <number>1</number>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QTableView" name="tvRightMatrix"/>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QLabel" name="lbEqual">
      <property name="font">
       <font>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="text">
       <string>=</string>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_7">
        <item>
         <widget class="QPushButton" name="pbMult">
          <property name="text">
           <string>Умножить</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pbReset">
          <property name="text">
           <string>Сброс</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QTableView" name="tvResultMatrix"/>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
